components/reply.js

```diff
@@ -173,7 +174,7 @@ export default forwardRef(function Reply ({
       {reply &&
         <div className={styles.reply}>
           <FeeButtonProvider
-            baseLineItems={postCommentBaseLineItems({ baseCost: 1, comment: true, me: !!me })}
+            baseLineItems={postCommentBaseLineItems({ baseCost: bio ? 1 : sub.replyCost, comment: true, me: !!me })}
```

Comment:
the saloon is a weird item that has no sub, so we need to check for sub here:

```suggestion
            baseLineItems={postCommentBaseLineItems({ baseCost: bio || !sub ? 1 : sub.replyCost, comment: true, me: !!me })}
```

Reply:
ah, I totally forgot saloon, applied!